daffodil-test/src/test/resources/org/apache/daffodil/section07/variables/variables.tdml

```diff
@@ -802,7 +802,7 @@
     <tdml:errors>
       <tdml:error>Schema Definition Error</tdml:error>
       <tdml:error>newVariableInstances must all be distinct within the same scope</tdml:error>
-      <tdml:error>ex:myVar1</tdml:error>
+      <tdml:error>myVar1</tdml:error>
     </tdml:errors>
```

Comment:
We should probably also add some tests for the Java and Scala API's to make sure that people using those API's can actually use this API. I suspect we'll need new wrapper classes for DaffodilXMLReader for japi and sapi like we have for DataProcessor.